modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/DateTimeObjectFieldBusinessType.java

```diff
@@ -229,10 +229,11 @@ private Timestamp _getTimestamp(
 			return null;
 		}
 
+		String sourceTimeZoneId = value.endsWith("Z") ? StringPool.UTC :
```

Comment:
I don't think this is a proper solution. I am thinking about different use cases not covered by this. For example, sending a different zimeone than Z